src/commcare_cloud/environment/secrets/secrets.yml

```diff
@@ -166,3 +166,5 @@
   legacy_namespace: localsettings_private
 - name: ansible_sudo_pass
   required: yes
+- name: DETAILED_TAGGING_DOMAINS
```

Comment:
@dannyroberts is the new naming convention all lower case (e.g., `ansible_sudo_pass`) or is that an exception to the rule?

Reply:
I believe the convention is to match where it is used. So, lowercase for these formplayer settings since they are lowercase in the formplayer config, and uppercase for the python localsettings, since they are all uppercase python constants.

Reply:
Thanks @calellowitz